latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -35,7 +35,7 @@ Version 2.0.13
 
 ### Component
 
-* Added deprecation warning to 'az component' commands
+* Deprecated all `component` commands
```

Comment:
The commands are still there but there was a warning added saying they'll be deprecated in the future.
Reading this, it looks like they *have been* deprecated already.